src/altinn-app-frontend/src/selectors/getErrors.ts

```diff
@@ -1,13 +1,23 @@
 import { IRuntimeState } from 'src/types';
 import { createSelector } from 'reselect';
 
+let lastLogged:Error|null = null;
+
 /**
  * Selector for determining if we have an error in one of our api calls.
  * Returns true any errors is set in relevant states, false otherwise
  * @param state the redux state
  */
 const getHasErrorsSelector = (state: IRuntimeState) => {
-  const error = (
+  const exceptIfIncludes = (maybeError:Error|null, lookFor:string):Error|null => {
```

Comment:
Just a note on the selector here in general, it appears we are not using reselect correctly. We are doing the calculation in the inputSelector and not the resultFunc, from docs:

`createSelector(...inputSelectors | [inputSelectors], resultFunc, selectorOptions?)
` (https://github.com/reduxjs/reselect#api)

See that this is not something you introduced. I'm guessing we probably have more issues like this.

Reply:
Good catch, I missed this.

I think we have several cases of incorrect usage of reselect - perhaps we should create a separate task to clean them up?